src/api/java/com/ldtteam/structurize/api/util/Utils.java

```diff
@@ -49,4 +52,31 @@ public static void checkDirectory(final File directory)
             Log.getLogger().error("Directory doesn't exist and failed to be created: " + directory.toString());
         }
     }
+
+    /**
+     * Check if two compound tags are equal.
+     * @param originTag the origin tag.
+     * @param compareTag the tag to compare.
+     * @param strictMatching strict matching.
+     * @return true if they match.
```

Comment:
Repeating the parameter names is not very good javadoc 😛 

From the doc alone I would have no idea what this does.  "origin" and "compare" are not very clear names.

Reply:
true the name is a bit generic, I'd like a name that clearly indicates the nature of the left-right comparison, maybe compareNBTofTo? or compareNBTLeftToRight
Just so that when you're using/reading it in code is clear this is not a normal strict equals